GetConfigFiles.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+import os
+import sys
+import argparse
+import json
+import pprint
+import mysql.connector
+
+
+def exec_sql(cursor, sql_stmt, params, error_msg):
+    """executes a simple sql statement"""
+    try:
+        cursor.execute(sql_stmt, params)
+    except Exception as err:
+        print(f"{str(err)} \n {error_msg}")
+        sys.exit()
+
+
+def exec_fetchall(cursor, sql_stmt, params, error_msg):
+    """executes the sql statmement and fetches all in a list"""
+    exec_sql(cursor, sql_stmt, params, error_msg)
+    try:
+        result = cursor.fetchall()
+    except Exception as err:
+        print(f"{str(err)} \n {error_msg}")
+        sys.exit()
+    return result
+
+
+def exec_fetchone(cursor, sql_stmt, params, error_msg):
+    """executes the sql stmt and fetches the first one in the result list"""
+    exec_sql(cursor, sql_stmt, params, error_msg)
+    try:
+        result = cursor.fetchone()
+    except Exception as err:
+        print(f"{str(err)} \n {error_msg}")
+        sys.exit()
+    if result:
+        return result[0]
+    else:
+        return None
+
+
+def write_file_from_db(database, script):
+    """As a work-a-round for RWM, share config files though the database"""
+    host = database["host"]
+    admin_acct = "root"
+    admin_pass = database["admin_password"]
+    # it is ok if the file doesn't as the clouds.yaml is possibly empty or manually updated
+    try:
+        cnx = mysql.connector.connect(host=host, user=admin_acct, password=admin_pass)
+    except mysql.connector.Error as err:
+        print(str(err))
+        sys.exit()
+    print(f"write {script} file_share_db")
+    cursor = cnx.cursor()
+
+    print(f"select file_name, file_data from file_share_db.file where script='{script}'")
+    data = exec_fetchall(
+        cursor,
+        "select file_name, file_data from file_share_db.file where script=%s",
+        (script,),
+        "Unable to select file from db",
+    )
+    pprint.pprint(data)
```

Comment:
We should make debugging statements conditional on some sort of debug flag: we don't want to see this output all the time (because cluttered logs make it harder to spot real problems). Consider using the `logging` module along with appropriate log levels.

Reply:
I have removed many of the long ones.  I do however like to see checkpoints in the log file in order that I can follow the code as it is working, not just logging errors.  This way I have at least a decent chance of viewing a log file and understanding if it is behaving normally.